external-adapter/src/requester.ts

```diff
@@ -86,18 +87,17 @@ export class Requester {
     return new AdapterError({ jobRunID, statusCode, message: error }).toJSONResponse()
   }
 
-  static success(jobRunID = '1', response: AxiosResponse) {
-    if (!('result' in response.data)) {
```

Comment:
What would be the reason to remove this?

Reply:
The optional chaining on line 94 is more concise but if you find this more readable or there was a reason for `null` specifically can revert.

Reply:
Hmm... :) Disregard then.